tests/mocks/footnotes.ts

```diff
@@ -0,0 +1,11 @@
+import merge from 'lodash/merge';
+
+const mockFootnoteKhattabSurah1Verse2 = (overrides = {}) =>
+  merge(
+    {
+      text: 'i.e., Lord of everything in existence including angels, humans, and animals.',
```

Comment:
Ditto: let's try to keep the mocked data as close to the actual API response as possible.